src/components/Explore/hooks/useInfiniteExploreScroll.js

```diff
@@ -96,8 +98,16 @@ const useInfiniteExploreScroll = ( { params: newInputParams, enabled }: Object )
     await refetch( );
   };
 
-  const observations = flatten( data?.pages?.map( r => r.results ) ) || [];
+  let observations = flatten( data?.pages?.map( r => r.results ) ) || [];
   let totalResults = data?.pages?.[0].total_results;
+
+  // filter out obs from excluded user and adjust count
+  if ( excludedUser && observations ) {
```

Comment:
I think this code is causing a bunch of rerenders in certain circumstances. If I filter by excluding the current user, I'm seeing the screen rerender over and over again with a different number of species and different results. If I then search by a different taxon, the screen rerenders 4-5 times, showing a blank screen in between each time.